### DISCUSSION TITLE
Reading ms office files content

### DISCUSSION THREAD

**Discussion by deepanshug** [winfsp/winfsp#453] [Q&A] [answered]
Hi, 

I am trying to read the content of docx file when opened from mount point. Read file API returns me the binary data which is not understood by docx file and it complains the file is corrupted. I even tried using another library like "Aspose" but that causes sort of infinite loop.

Is there any way to read ms office files. e.g. docx.

thanks
Deepanshu

**Comment by billziss-gh** [comment, marked as answer]
Your question is not clear. Do you have a file system that hosts a DOCX file that cannot be read by Office?

Office definitely works correctly with WinFsp.

**Reply by deepanshug** [reply]
This was my mistake where I was corrupting the data.
I am sorry for confusion.
